src/jdk.incubator.jextract/share/classes/jdk/internal/clang/LibClang.java

```diff
@@ -31,11 +31,12 @@
 
 public class LibClang {
     private static final boolean DEBUG = Boolean.getBoolean("libclang.debug");
+    private static final boolean NO_CRASH_RECOVERY = Boolean.getBoolean("libclang.disable_crash_recovery");
 
     public static Index createIndex(boolean local) {
         Index index = new Index(Index_h.clang_createIndex(local ? 1 : 0, 0));
-        Index_h.clang_toggleCrashRecovery(0);
-        if (DEBUG) {
+        Index_h.clang_toggleCrashRecovery(NO_CRASH_RECOVERY ? 0 : 1);
```

Comment:
I vaguely recall a specific reason for this default. Maurizio may recall about this